bot/exts/recruitment/talentpool/_review.py

```diff
@@ -326,10 +327,12 @@ async def archive_vote(self, message: PartialMessage, passed: bool) -> None:
         result = f"**Passed** {Emojis.incident_actioned}" if passed else f"**Rejected** {Emojis.incident_unactioned}"
         colour = Colours.soft_green if passed else Colours.soft_red
         timestamp = datetime.now(tz=UTC).strftime("%Y/%m/%d")
+        thread_jump = f"[Jump to vote thread]({thread.jump_url})" if thread else "Failed to get thread"
```

Comment:
You technically don't need to care about getting the thread or failing to do so. The thread URL is just the message URL without the message ID. I think it will be more consistent that way, because otherwise the thread might not be cached and then you have to do `await guild.fetch_channel`. This is unlikely to come up in testing since you're probably archiving recently created threads.

Alternatively, going further down the function there's a definition for `nomination_thread`, which you can just move up to above its new earliest usage (here).

Reply:
Good point. I made it use the existing variable: https://github.com/python-discord/bot/pull/2536/commits/cf8409463390a2bdb03720bed912ff61a36fa4ae